src/visualization/types/Map/GeoOptions.tsx

```diff
@@ -36,6 +39,27 @@ export enum MapType {
 
 export const GeoOptions: FC<Props> = ({properties, update, results}) => {
   const {useS2CellID} = properties
+
+  if (!properties.layers.length) {
```

Comment:
`!properties?.layers?.length` is probably better unless you can guarantee `properties.layers` always exists here